### PR TITLE
JKNS-569: Update Image for CI Testing

### PR DESCRIPTION
**Pull request by prdhamdh**
debugging CI test failure

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,7 +13,7 @@ RUN curl -L -o maven.tar.gz https://dlcdn.apache.org/maven/maven-3/3.8.8/binarie
 RUN ./maven/bin/mvn --version && \
 	./maven/bin/mvn clean package
 
-FROM registry.ci.openshift.org/origin/4.16:jenkins
+FROM registry.redhat.io/ocp-tools-4/jenkins-rhel8:v4.14.0
```
Comment (adambkaplan) [inline Dockerfile:16]: Let's just go ahead and merge this! Green CI is good CI.

**Review by adambkaplan — APPROVED** [review]
/approve

**Comment by adambkaplan** [comment]
/retitle JKNS-569: Update Image for CI Testing

**Comment by adambkaplan** [comment]
@prdhamdh if you can update your commit message so it describes the change made, then this should be ready to be merged once it is approved.

**Review by adambkaplan — APPROVED** [review]
/lgtm

The `security` test can be ignored - it is an "informative" test that really should be disabled for good.

**Comment by prdhamdh** [comment]
/cherry-pick release-4.18 release-4.17 release-4.16 release-4.15 release-4.14 release-4.13 release-4.12

**Pull request merged**